core/os/input_event.cpp

```diff
@@ -210,11 +245,16 @@ void InputEventWithModifiers::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_command", "enable"), &InputEventWithModifiers::set_command);
 	ClassDB::bind_method(D_METHOD("get_command"), &InputEventWithModifiers::get_command);
 
+	ClassDB::bind_method(D_METHOD("set_action_pressed_on_modifier", "enable"), &InputEventWithModifiers::set_action_pressed_on_modifier);
+	ClassDB::bind_method(D_METHOD("is_action_pressed_on_modifier"), &InputEventWithModifiers::is_action_pressed_on_modifier);
+
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "alt"), "set_alt", "get_alt");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "shift"), "set_shift", "get_shift");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "control"), "set_control", "get_control");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "meta"), "set_metakey", "get_metakey");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "command"), "set_command", "get_command");
+
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "isActionPressedOnModifier"), "set_action_pressed_on_modifier", "is_action_pressed_on_modifier");
```

Comment:
You're a long time Godot user, you might have figured by now that all properties are `snake_case` ;)

Reply:
And the property should be `action_pressed_on_modifier`, not with `is_` prefix (that's the getter).

That being said, this is *probably* not a wanted addition, but just mentioning the style issue in any case.